targets/javascript/testingFiles/PlayFabApiTest.ts

```diff
@@ -207,35 +206,6 @@ var PlayFabApiTests = {
         assert.ok(typeof loginPromise.then === "function" && Promise.resolve(loginPromise) === loginPromise, "Testing whether the login request returned a promise object");
     },
 
-    /* CLIENT API
-     * Test that the login call sequence sends the AdvertisingId when set
-     */
-    LoginWithAdvertisingId: function (assert): void {
```

Comment:
Javascript doesn't run automated tests, so you'll have to load the test-web-page manually. Please do that.